### ISSUE TITLE
One-Time charge

### ISSUE THREAD

**Issue by divdax** [mollie/laravel-cashier-mollie#120]
I try to implement the one-time charge with code from the documentation: https://www.cashiermollie.com/04-charges.html

```
Call to undefined method Laravel\Cashier\Charge\MandatedChargeBuilder::setRedirectUrl()
```
Could you explain what is this for? Is this redirect get called when a payment was successful?
```
return back()->with('status', 'Thank you.');
```

**Comment by sandervanhooft** [comment]
Thanks @divdax for bringing this to our attention.

Looks like it's a bug - I've prepared a fix here: #121 


To answer your question: After leaving Mollie's checkout (payment success/cancel/failure) the customer is returned to the provided `redirectUrl`. With subscriptions and charges this only happens on the first payment, when no valid payment mandate is present.

**Comment by divdax** [comment]
Great! Thank you @sandervanhooft for the fix.
